src/opt-out.html

```diff
@@ -39,10 +39,10 @@ <h1 class="govuk-heading-xl">
         How to opt out of emergency alerts
       </h1>
       <p class="govuk-body">
-        If you opt out, you will not get any emergency alerts.
+        You can opt out of some emergency alerts, but not the most important ones.
       </p>
       <p class="govuk-body">
-        You cannot choose the type of hazard you want to get alerts about. If you opt out because you do not want flood warnings for example, you’ll also miss alerts for fires and terrorist incidents.
+        You cannot opt out by subject, only by how serious the emergency is. If you opt out because you do not want flood warnings, for example, you’ll also miss alerts for fires and terrorism.
```

Comment:
```suggestion
        You cannot opt out by subject, only by how serious the emergency is. If you opt out because you do not want flood warnings, for example, you might miss alerts for fires and terrorism.
```